swap_meet/vendor.py

```diff
@@ -1,2 +1,62 @@
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, inventory_item):
+        self.inventory.append(inventory_item)
+        return inventory_item
+
+    def remove(self, inventory_item):
+        if inventory_item in self.inventory:
+            self.inventory.remove(inventory_item)
+            return inventory_item
+        else:
+            return False
+
+    # inventory is a list of item istances with each item having a category as an attribute
+    # get_by_category filters inventory by category
+    # items is a list of just the filtered item instances
+    def get_by_category(self, category):
```

Comment:
This would be a great spot to use list comprehension!

```
same_category = [item for item in self.inventory if item.category == category]
```

Here's more examples of Python list comprehensions if you want to continue playing around with them 😄 : https://www.w3schools.com/python/python_lists_comprehension.asp